cmd/model/write.go

```diff
@@ -57,7 +57,30 @@ var writeCmd = &cobra.Command{
 			return fmt.Errorf("failed to initialize FGA Client due to %w", err)
 		}
 
-		response, err := write(fgaClient, args[0])
+		fileName, err := cmd.Flags().GetString("file")
+		if err != nil {
+			return fmt.Errorf("failed to parse file name due to %w", err)
+		}
+
+		var inputModel string
+		if fileName != "" {
+			file, err := os.ReadFile(fileName)
+			if err != nil {
+				return fmt.Errorf("failed to read file %s due to %w", fileName, err)
+			}
+			inputModel = string(file)
+		} else {
+			if len(args) == 0 || args[0] == "-" {
+				err := cmd.Help()
+				if err != nil {
+					return fmt.Errorf("failed to print help message due to %w", err)
+				}
+				os.Exit(0)
```

Comment:
why do we have os.Exit(0)?  would return cmd.Help() be sufficient?

Reply:
or do we want to have os.Exit(1)?

Reply:
fixed